sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -902,33 +902,45 @@ case class SortArray(base: Expression, ascendingOrder: Expression)
 
 
 /**
- * Sorts the input array in ascending order according to the natural ordering of
+ * Sorts the input array in ascending / descending order according to the natural ordering of
  * the array elements and returns it.
  */
-// scalastyle:off line.size.limit
+// scalastyle:on line.size.limit
```

Comment:
This looks incorrect, unless the line is now not too long, but it seems to be longer